checkQC/config.py

```diff
@@ -1,8 +1,30 @@
 
+from pkg_resources import Requirement, resource_filename
+
 import yaml
 
 
 def get_config(config_file):
-    with open(config_file) as stream:
-        config = yaml.load(stream)
-    return config
+    try:
+        if not config_file:
+            config_file = resource_filename(Requirement.parse('checkQC'), 'checkQC/default_config/config.yaml')
+            print("No config file specified, using default config from {}.".format(config_file))
+
+        with open(config_file) as stream:
+            config = yaml.load(stream)
+        return config
```

Comment:
I believe you can return from within the with statement. One line of code removed!